src/styles/components/index.scss

```diff
@@ -0,0 +1,33 @@
+.home {
+  position: relative;
+}
+
+.home-top {
+  padding: 13rem 0.5rem 0 8rem;
+  max-width: 47%;
```

Comment:
Quick question: [47% feels a little magic](https://en.wikipedia.org/wiki/Magic_number_(programming)). Where's it coming from?